packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -121,12 +121,6 @@ export async function createLanguageServer(
   context: vscode.ExtensionContext
 ): Promise<LanguageClient> {
   const isInsiders: boolean = /insiders/i.test(vscode.env.appName);
```

Comment:
You also need to remove the isInsiders variable.

Reply:
@lcampos done.